charts/portal/values.yaml

```diff
@@ -399,8 +399,22 @@ backend:
     frameDocumentTypeIds:
       type0: "CX_FRAME_CONTRACT"
     onboardingServiceProvider:
-      # -- Client-secret for onboardingserviceprovider encryptionKey. Secret-key 'onboardingserviceprovider-encryption-key'.
-      encryptionKey: ""
+      encryptionConfigIndex: 1
+      encryptionConfigs:
+        index0:
+          index: 0
+          cipherMode: "ECB"
+          paddingMode: "PKCS7"
+          # -- Client-secret for onboardingserviceprovider encryptionKey. Secret-key 'onboardingserviceprovider-encryption-key0'.
+          # -- expected format is 256 bit (64 digits) hex.
```

Comment:
Please remove "--", the comment is already started in L408, and camelcase "Expected". Same for other occurrences.

Reply:
done.